### PR TITLE
add csi-secrets-store periodic, postsubmit to sig-auth

### PR DESCRIPTION
**Pull request by aramase**
Signed-off-by: Anish Ramasekar <anish.ramasekar@gmail.com>

- Adds `sig-auth-secrets-store-csi-driver-postsubmit`, `sig-auth-secrets-store-csi-driver-periodic`, `sig-auth-secrets-store-csi-driver-presubmit` and `sig-auth-secrets-store-csi-driver-release-signal` to `sig-auth` dashboard group

/assign @ritazh

### CODE REVIEW

File: config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ presubmits:
         - make
         - test-style
     annotations:
-      testgrid-dashboards: sig-auth-secrets-store-csi-driver
+      testgrid-dashboards: sig-auth-secrets-store-csi-driver, sig-auth-secrets-store-csi-driver-presubmit
```
Comment (ritazh) [inline config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml:21]: > Removes sig-auth-secrets-store-csi-driver from dashboard group

Did you want to remove it?

File: config/testgrids/config.yaml [hunk 2]
```diff
@@ -250,6 +250,10 @@ dashboards:
 - name: sig-big-data
 - name: sig-multicluster-kubemci
 - name: sig-auth-secrets-store-csi-driver
```
Comment (ritazh) [inline config/testgrids/config.yaml:252]: same here

File: config/testgrids/config.yaml [hunk 3]
```diff
@@ -298,3 +302,7 @@ dashboard_groups:
   dashboard_names:
   - sig-auth-gce
   - sig-auth-secrets-store-csi-driver
```
Comment (ritazh) [inline config/testgrids/config.yaml:304]: and here

File: config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml (hunk 1 above)
Comment (aramase) [reply config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml:21]: I updated the PR description. It's not possible to remove it. All the dashboards need to part of the `dashboard_group`

Comment (aramase) [reply config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml:21]:
```
--- FAIL: TestConfig (0.12s)
    config_test.go:340: Dashboard sig-auth-secrets-store-csi-driver should be in dashboard_group sig-auth
```
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/22162/pull-test-infra-bazel/1392579412947374080

Comment (ritazh) [reply config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml:21]: Gotcha. We do want to see status of all the jobs. With this PR we will see them in groups based on their types and one with all the jobs in it.

Comment (aramase) [reply config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml:21]: That's right!

**Comment by aramase** [comment]
@spiffxp @shyamjvs Could you take a look at this PR when you get a chance?

**Comment by aramase** [comment]
@spiffxp Could you help review this PR?

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm
Sorry I haven't seen this until just now

**Comment by aramase** [comment]
Thank you @spiffxp!

**Pull request merged**